client/types/account-overview.d.ts

```diff
@@ -1,60 +1,63 @@
-declare namespace AccountOverview {
```

Comment:
Inline with the advice from Slack (p1680732280411199/1680681067.811289-slack-C045PC0V40G) this PR dissolves the `AccountOverview` namespace in favour of exported interfaces. 

Client files of the AccountOverview namespace have been updated to import the interfaces like this:

```php
import * as AccountOverview from 'wcpay/types/account-overview';
```